### ISSUE TITLE
Prettier configuration breaks my code with a parsing error

### ISSUE THREAD

**Issue by polomoshnov** [react-boilerplate/react-boilerplate#2853]
## Description
I've stumbled upon a very strange error when I was porting my existing codebase to React Boilerplate. The matter is that some of my Thunk functions got broken after I run `npm run lint:eslint:fix -- .` in order to autofix this codebase that was initially written with the help of CRA. I suspect the error comes from Prettier or the way it is intergrated with React Boilerplate.

## Steps to reproduce
If you're on Windows run this in the console:
```
git clone https://github.com/react-boilerplate/react-boilerplate.git && ^
cd react-boilerplate && ^
rmdir /Q /S app && ^
mkdir app && ^
cd app && ^
curl -L -o notSoLongActionName.js https://gist.github.com/polomoshnov/a29b669c40b062330d1dee716e0d3da3/raw/d2d3549bc8a2957a5e27be58ccac10e5a933929a/notSoLongActionName.js && ^
curl -L -o quiteLongActionName.js https://gist.github.com/polomoshnov/a29b669c40b062330d1dee716e0d3da3/raw/d2d3549bc8a2957a5e27be58ccac10e5a933929a/quiteLongActionName.js && ^
npm install && ^
npm run lint:eslint:fix -- .
```
Those [two gist files](https://gist.github.com/polomoshnov/a29b669c40b062330d1dee716e0d3da3) are almost alike except that the name in `quiteLongActionName.js` is longer than in `notSoLongActionName.js` by an extra `o` char. 

So, at the end of `npm run lint:eslint:fix -- .` you get `quiteLongActionName.js` with a parsing error while `notSoLongActionName.js` doesn't have any parsing error.

**Expected behavior**
`npm run lint:eslint:fix -- .` should not produce `quiteLongActionName.js` with a parsing error just because it contains a function name that is one character longer than in `notSoLongActionName.js` and the rest of the code is the same in both files.

**Screenshots**
![err](https://user-images.githubusercontent.com/1019132/73837440-35344d80-485d-11ea-9a1b-07f43c0eaaf9.png)

## Versions

- React-Boilerplate: latest
- Node/NPM: latest
- Browser: latest

**Comment by vineyardbovines** [comment]
I'm a bit confused by this -- the screenshot does contain a parsing error?

**Comment by polomoshnov** [comment]
Yes, it contains. And if you git clone React Boilerplate and add my two files to it and then run `npm run lint:eslint:fix -- .` you will see for yourself that only one of the files gets corrupted although both of them are identical with the only difference:
```js
// in notSoLongActionName.js this function name has fewer chars
loooooooooooooooooooooooooooooooooooooooooooooooooooooongAction()
// than in quiteLongActionName.js
looooooooooooooooooooooooooooooooooooooooooooooooooooooongAction()
```

**Comment by polomoshnov** [comment]
I think this issue has something to do with `printWidth` in [.prettierrc](https://github.com/react-boilerplate/react-boilerplate/blob/master/.prettierrc) because if you try to increase it, for example, to 81 the issue is gone.

And also, if you don't change `printWidth` but only format the code with Prettier before fixing it with linter you get no error either: `npm run prettify -- "app/**/*.js" && npm run lint:eslint:fix -- .`

**Comment by polomoshnov** [comment]
I just found out from [here](https://github.com/prettier/eslint-config-prettier#special-rules) that one might end up with invalid code when [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier), [eslint-plugin-prettier](https://github.com/prettier/eslint-plugin-prettier) and `--fix` are used all together. The following is said there:

>  you need to keep in mind that you might end up with invalid code, where you manually have to insert a missing closing parenthesis to get going again.
> If you’re fixing large amounts of previously unformatted code, consider temporarily disabling the prettier/prettier rule and running eslint --fix and prettier --write separately.

So, perhaps that's the bug we're facing here. And exactly my usecase is mentioned there: ` fixing large amounts of previously unformatted code`.

**Comment by polomoshnov** [comment]
Maybe React Boilerplate should warn people about this issue somehow or handle it on npm scripts level, `running eslint --fix and prettier --write separately`?
